<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalculatriceIP</class>
 <widget class="QMainWindow" name="CalculatriceIP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CalculatriceIP</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Adresse IPv4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Suffixe</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Masque</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre maximum de sous réseaux</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre maximum d'adresse IP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Adresse réseau</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Adresse IP utilisables</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Adresse de diffusion</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_adresse1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>192</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_adresse2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>168</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_adresse3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_adresse4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_adr_reseau">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>160</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_adr_ip">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>200</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_suffixe">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_masque">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_sous_reseau_max">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_max_ip">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCalculer">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonQuitter">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>310</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_adr_diffusion">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>250</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPropos">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>310</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>A propos</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>499</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCalculatrice_IPv4">
    <property name="title">
     <string>Calculatrice IPv4</string>
    </property>
   </widget>
   <addaction name="menuCalculatrice_IPv4"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonQuitter</sender>
   <signal>clicked()</signal>
   <receiver>CalculatriceIP</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>382</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_masque</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>comboBox_suffixe</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>454</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_suffixe</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>comboBox_masque</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>458</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_masque</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>comboBox_sous_reseau_max</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_sous_reseau_max</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>comboBox_masque</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_sous_reseau_max</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>comboBox_max_ip</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_max_ip</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>comboBox_sous_reseau_max</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_masque</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>comboBox_max_ip</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_max_ip</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>comboBox_masque</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_sous_reseau_max</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>comboBox_suffixe</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>457</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_suffixe</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>comboBox_sous_reseau_max</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>394</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_max_ip</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>comboBox_suffixe</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>462</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_suffixe</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>comboBox_max_ip</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>455</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
